src/components/userDasboard/components/DashboardDetails.jsx

```diff
@@ -35,29 +36,29 @@ const DashboardDetails = ({ numberOfOrgs, numberOfDictionary, organizations }) =
               </a>
             </span>
           ))}{' '}
+          <span className="d-block">
           This can be changed via{' '}
-          <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
+            <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
             the traditional OCL
-          </a>.
-        </p>
-        <p className="lead">
-          You can <Link to="dashboard/dictionaries">view all OpenMRS public dictionaries</Link>.
+            </a>.
+          </span>
         </p>
       </div>
     );
   }
   return (
     <div className="user-data">
-      <p className="lead">
+      <span>
+        <p className="lead">
         You do not belong to any organization. This can be changed via{' '}
-        <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
+          <a href="https://qa.openconceptlab.org/" target="_blank" rel="noopener noreferrer">
           the traditional OCL
-        </a>.
-      </p>
-      <p className="lead">
-        You currently have {numberOfDictionary} personal {dictionary}. You can{' '}
-        <Link to="dashboard/dictionaries">view all OpenMRS public dictionaries</Link>.
```

Comment:
Isn't this a change in the UI?

Reply:
It is UI change. When I rebased, I picked this changes from alexmochu PR who was working on this changes